packages/terra-clinical-data-grid/src/terra-dev-site/doc/clinical-data-grid/DatagridExamples.2/DatagridWithCustomContent.scss

```diff
@@ -0,0 +1,12 @@
+@import './Datagrid';
```

Comment:
IMO these docs styles can be merged into a common example with class names that are meaningful to their intent